src/fu-util-common.c

```diff
@@ -2445,6 +2445,26 @@ fu_util_security_event_to_string(FwupdSecurityAttr *attr)
 		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
 		      /* TRANSLATORS: HSI event title */
 		      _("UEFI memory protection is now unlocked")},
+		     /* ------------------------------------------*/
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_UNKNOWN,
+		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
+		      /* TRANSLATORS: HSI event title */
+		      _("AMD microcode signature vulnerability detected")},
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_UNKNOWN,
+		      FWUPD_SECURITY_ATTR_RESULT_LOCKED,
+		      NULL},
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
+		      FWUPD_SECURITY_ATTR_RESULT_LOCKED,
+		      /* TRANSLATORS: HSI event title */
+		      _("AMD microcode signature vulnerability fixed")},
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_LOCKED,
+		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
+		      /* TRANSLATORS: HSI event title */
+		      _("AMD microcode signature vulnerability broken")},
```

Comment:
How about `AMD microcode signature not trusted`?

Reply:
This is for the (perhaps impossible) case where the user downgrades the firmware so that they went from fixed to unfixed.